assets/workflow/Immunobiology/Immunobiology.protocol 5.html

```diff
@@ -1,85 +1,123 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
-<head>
-<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-<title>Untitled Document</title>
-<style>
-img {
-    display: block;
-    margin-left: auto;
-    margin-right: auto;
-}
-</style>
-</head>
+<html>
+  <head>
+    <title>Untitled Document</title>

```

Comment:
You can use this attribute to mark the 'title' of this html content(it is not a real html title since it will be embeded in our real resulting html FYI) if you want to make it easier identify which html is for which 'page' since not all subtitles and the headers are exactly matched.
If you don't need it you can just remove this attribute since Untitled Document is useless here.
Same in other html files.

Reply:
Good Idea, will do that